addons/l10n_ec/models/account_tax.py

```diff
@@ -20,6 +35,11 @@ class AccountTax(models.Model):
         string="Code ATS",
         help="Tax Identification Code for the Simplified Transactional Annex",
     )
+    l10n_ec_code_taxsupport = fields.Selection(
+        L10N_EC_TAXSUPPORTS,
+        string='Tax Support',
+        help='Indicates if the purchase invoice supports tax credit or cost or expenses, conforming table 5 of ATS'
+    )

```

Comment:
:warning: new field, cannot be added in stable :see_no_evil:

Reply:
Field moved to enterprise in v16
Still to do: script to move it back to community in master